apps/full-site-editing/full-site-editing-plugin/common/index.js

```diff
@@ -1,4 +1,52 @@
+/**
+ * External dependencies
+ */
+import { PluginSidebar, PluginSidebarMoreMenuItem } from '@wordpress/edit-post';
+import { layout, plus } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { registerPlugin } from '@wordpress/plugins';
+import { PanelBody, Button } from '@wordpress/components';
+
 /**
  * Internal dependencies
  */
 import './index.scss';
+
+const BlockPatternsMoved = () => {
+	const openInserter = () => {
+		const inserterButton = document.querySelector( '.edit-post-header-toolbar__inserter-toggle' );
+		inserterButton && inserterButton.click();
+		setTimeout( () => {
+			const patternsButton = document.querySelector(
+				'.block-editor-inserter__tabs [id$="patterns"]'
+			);
+			patternsButton && patternsButton.click();
+		}, 300 );
+	};
+
+	return (
+		<>
+			<PluginSidebarMoreMenuItem icon={ layout } target="block-patterns-moved">
+				{ __( 'Block Patterns Have Moved' ) }
+			</PluginSidebarMoreMenuItem>
+			<PluginSidebar
+				icon={ layout }
+				name={ 'block-patterns-moved' }
+				title={ __( 'Block Patterns Have Moved' ) }
+				className="common__block-patterns-moved"
+			>
+				<PanelBody>
+					<h2>{ __( 'Block Patterns have moved!' ) }</h2>
+					<p>
+						{ __(
+							'You can now find them in the block inserter under the "patterns" tab.  The block inserter can be found in the top-left corner of your screen by selecting the following icon:'
+						) }
+					</p>
+					<Button isPrimary icon={ plus } onClick={ openInserter } />
+				</PanelBody>
+			</PluginSidebar>
+		</>
+	);
+};
+
+registerPlugin( 'block-patterns-moved', { render: BlockPatternsMoved } );
```

Comment:
Both `plus` and `layout` icons were added in Gutenberg v7.6 (https://github.com/WordPress/gutenberg/pull/20234, https://github.com/WordPress/gutenberg/pull/20181) and patterns were added in v7.7. It's not exact but we could just not register this if those icons are `undefined`? Those users never had patterns to beging with.

Will avoid fatals in older Gutenbergs on Atomic anyway.

Reply:
`icons` is unique, it's bundled rather than external.

Reply:
Ahh right, forgot about that. So then that doesn't matter, it's not a biggie to ship this to older versions. Will just increase awareness. :-)